releases/README.md

```diff
@@ -62,13 +63,15 @@
 - [Release 1.2.2](https://github.com/skalenetwork/skale-node/releases/tag/1.2.2) (27 Apr 2021)
 - [Release 1.2.3](https://github.com/skalenetwork/skale-node/releases/tag/1.2.3) (18 June 2021)
 - [Release 2.0.0](https://github.com/skalenetwork/skale-node/releases/tag/2.0.0) (20 July 2021)
+- [Release 2.0.2](https://github.com/skalenetwork/skale-node/releases/tag/2.0.2) (24 Sept 2021)
 
 #### Skaled
 
 - [Release 1.46-stable.0](https://github.com/skalenetwork/skaled/releases/tag/1.46-stable.0) (17 Sept 2020)  
 - [Release 3.5.12-stable.1](https://github.com/skalenetwork/skaled/releases/tag/3.5.12-stable.1) (19 Apr 2021) 
 - [Release 3.5.14-stable.0](https://github.com/skalenetwork/skaled/releases/tag/3.5.14-stable.0) (18 June 2021) 
 - [Release 3.7.3-stable.0](https://github.com/skalenetwork/skaled/releases/tag/3.7.3-stable.0) (20 July 2021) 
+- Release 3.7.5-stable.0 (24 Sept 2021)
```

Comment:
we don't have a stable release of skaled on Github?

Reply:
Yes, there were problems with build

Reply:
@dmitry-tk  yes, we are releasing container for now